Source/Extensions/Blazorise.TreeView/TreeView.razor.cs

```diff
@@ -58,6 +57,39 @@ await foreach ( var nodeState in paramNodes.ToNodeStates( HasChildNodesAsync, Ha
                 treeViewNodeStates.Add( nodeState );
             }
         }
+
+        if ( paramNodes is INotifyCollectionChanged observableCollection )

```

Comment:
Great, seems like it's easy to support Obesrvables, we should do the same for DataGrid. Try for 1.3?

Reply:
Why not.